tests/test_cli.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of cernopendata-client.
+#
+# Copyright (C) 2020 CERN.
+#
+# cernopendata-client is free software; you can redistribute it and/or modify
+# it under the terms of the GPLv3 license; see LICENSE file for more details.
+
+"""cernopendata-client text user interface(tui) test."""
```

Comment:
This test suite will grow, so please create three test files, one for each CLI command:

- `tests/test_get_record.py`
- `tests/test_get_file_locations.py`
- `tests/test_download_files.py`

?

Reply:
Creating three test files sounds good, I was thinking of one more folder like

- `tests/tests_cli/test_get_record.py`
- `tests/tests_cli/test_get_file_locations.py`
- `tests/tests_cli/test_download_files.py`

which one looks fine?

Reply:
Yes, that's also OK, especially if we are going to split `cli.py` into `cli/command1.py`, `cli/command2.py` one day...